### DISCUSSION TITLE
Looks powerful, hoping it becomes business viable

### DISCUSSION THREAD

**Discussion by marchhenriksen** [platformplatform/PlatformPlatform#185] [General]
Hi Thomas

Given my low technical expertise, I can't say that I have completely understood what this aims to accomplish :)

However, if your intent is to provide a simple & powerful abstraction layer between infrastructure+basic services and the business, I do believe there is business potential for this.

I see this as doing to businesses, what AWS originally did for e-commerce. If done right, this should scale exceptionally well.

I would encourage you to also describe your platformplatform in terms that business people can understand, and probably also visually (for instance: it's a box that you connect your data & business rules to, and it handles the complicated parts of your backend & infrastructure, and then a simple drawing of that).

Best regards,
Marc

**Comment by tjementum** [comment]
Thanks Marc,

GitHub is primarily targeting developers, so yes it's technical. I plan to set up a public website, with a more business-minded presentation.

It's still very early, so it will take some time to make it presentable... but I will remember to ask for your feedback, when I have something more high-level. 😄

**Reply by marchhenriksen** [reply]
Very cool, let me know when I can help 🙂
